### ISSUE TITLE
Model Description

### ISSUE THREAD

**Issue by StephanSchaller** [Open-Systems-Pharmacology/Alfentanil-Model#7]
ToDo: The Initial Description in the Readme should contain a bullet-point list of base model specifics (metabolic pathways, drug specifics (can it be used as a reference scenario drug for a drug class?, IV / oral, specifics on formulation, etc...)

**Comment by Yuri05** [comment]
Reference to the original article is not enough? https://github.com/Open-Systems-Pharmacology/Alfentanil-Model#Reference

**Comment by PavelBal** [comment]
> Reference to the original article is not enough?

I don't think so. Reading a paper requires too much time just to understand what is covered in the model.

**Comment by StephanSchaller** [comment]
A concise summary should (ideally) be available. This is an extra 5 min of work for the person who developed the model, but would take half an hour for everyone interested in the model to extract from the paper...

**Comment by Yuri05** [comment]
This is now all included in the model evaluation plan.
